model/src/order.rs

```diff
@@ -614,44 +634,50 @@ mod tests {
         assert_eq!(serialized, value);
     }
 
-    // these two signature tests have been created by printing the order and signature information
     // from the test `should recover signing address for all supported ECDSA-based schemes` in
-    // https://github.com/gnosis/gp-v2-contracts/blob/main/test/GPv2Signing.test.ts .
+    // <https://github.com/gnosis/gp-v2-contracts/blob/v1.0.0/test/GPv2Signing.test.ts#L280>.
```

Comment:
Why not use [v1.0.1](https://github.com/gnosis/gp-v2-contracts/releases/tag/v1.0.1)